lib/net/nvnetdrv/nvnetdrv.c

```diff
@@ -201,7 +201,7 @@ static void nvnetdrv_handle_rx_irq (void)
             goto release_packet;
         }
 
-        if (!(flags & NV_RX_ERROR) || (flags & NV_RX_FRAMINGERR)) {
+        if (!(flags & NV_RX_ERROR) || ((flags & NV_RX_ERROR_MASK)) = NV_RX_FRAMINGERR)) {

```

Comment:
Considering that `NV_RX_SUBTRACT1` isn't part of `NV_RX_ERROR_MASK` either, wouldn't it make more sense to simplify the whole check to something like `if ((flags & NV_RX_CRIT_ERRORS) == 0)` with `#define NV_RX_CRIT_ERRORS (NV_RX_ERROR1|NV_RX_ERROR2|NV_RX_ERROR3|NV_RX_ERROR4|NV_RX_CRCERR|NV_RX_OVERFLOW)`, or similar?

Reply:
Good idea, done